subschemas/substrate-chain/src/typeDefs/treasury.ts

```diff
@@ -8,29 +8,20 @@ export default /* GraphQL */ `
     reservedBalance: String!
     votingBalance: String!
   }
-  type ProposalVotes {
-    index: String
-    threshold: String
-    ayes: [String!]
-    nays: [String!]
-    end: String
-  }
-  type CollectiveProposal {
-    hash: String!
-    votes: ProposalVotes!
-    callIndex: String!
-  }
-  type PalletProposal {
+
+  type TreasuryProposal {
+    index: String!
     proposer: AccountInfo!
     value: String!
     beneficiary: AccountInfo!
     bond: String!
   }
-  type TreasuryProposal {
-    councils: [CollectiveProposal!]!
-    id: String!
-    proposal: PalletProposal!
+
+  type Treasury {
```

Comment:
I'm not sure what the intention here is, but having this type `Treasury` is misleading. The treasury is a module, so should be the top most general type here